src/core/border_router/routing_manager.cpp

```diff
@@ -88,6 +91,9 @@ Error RoutingManager::Init(uint32_t aInfraIfIndex, bool aInfraIfIsRunning)
 
     SuccessOrExit(error = LoadOrGenerateRandomOmrPrefix());
     SuccessOrExit(error = LoadOrGenerateRandomOnLinkPrefix());
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_NAT64_ENABLE
```

Comment:
Add a Cmake option for configuration `OPENTHREAD_CONFIG_BORDER_ROUTING_NAT64_ENABLE` (e.g. `OT_BORDER_ROUTING_NAT64`) ?

Reply:
Added. Thanks!